ccf_specs.md

```diff
@@ -562,12 +588,19 @@ ccf-composite-type-message =
             / contract-type
             / event-type
             / enum-type
+            / struct-interface-type
+            / resource-interface-type
+            / contract-interface-type
         )
     ])
 
-composite-type = [
+composite-type-id = (
```

Comment:
Maybe rename this to just `type-id`, like it is used in Cadence

Reply:
Good idea!  Done in commit f9eba1dd61b6e867154b0b84942570a2c316831c.